Modules/Core/SpatialObjects/test/itkEllipseSpatialObjectTest.cxx

```diff
@@ -127,17 +127,19 @@ int itkEllipseSpatialObjectTest(int, char* [])
   }
   std::cout<<"[PASSED]"<<std::endl;
 
-  myEllipse->ComputeFamilyBoundingBox( EllipseType::MaximumDepth );
```

Comment:
@aylward I'm a little concerned that the ordering of these calls changes the behavior.  For example, calling "ComputeFamilyBoundingBox" must occur after calling "Update".

Should "ComputeFamilyBoundingBox" implicitly call update?

Reply:
As long as ComputeMyBoundingBox() has been called via Update(), there is no need for ComputeFamilyBoundingBox() to call Update().   We perhaps need to make it clear that function calls to get BoundingBoxes, PerimeterSize, Areas, and other values from the objects must occur after defining the parameters of the object and then calling Update().   I think this is a logical flow, and relatively easy to document.     My concern with calling Update() from ComputeFamilyBoundingBox() or any other function is that Update() could be a computationally expensive call.   We would need to make certain all computations within Update() are conditional on modified times, and they are not at this time.  One thing I did  not clear up in the refactoring is the odd handling of modified times by certain spatial objects - there are still some inconsistencies (and some pretzel logic, imo), but I think that the appropriate use of Update() mitigates those deficiencies and will not change behavior once they are resolved.